src/BlockchainReader/DAO.ts

```diff
@@ -24,25 +27,21 @@ export class DAO {
 
   readonly start = async (): Promise<void> => {
     await this.collection.createIndex({ blockHeight: 1 }, { unique: true })
+    await this.collection.createIndex({ blockHash: 1 }, { unique: true })
```

Comment:
`blockHeight` is not really unique, but we aren't dealing with that yet.

Still, to prepare for that, we start using `blockHash` instead.